nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Update/UpdateWorkerTests.PackagesConfig.cs

```diff
@@ -932,7 +932,15 @@ public async Task PackagesConfigUpdateCanHappenEvenWithMismatchedVersionNumbers(
                 realizedAdditionalFilesExpected.AddRange(additionalFilesExpected);
             }
 
-            await TestUpdateForProject(dependencyName, oldVersion, newVersion, projectContents, expectedProjectContents, additionalFiles: realizedAdditionalFiles.ToArray(), additionalFilesExpected: realizedAdditionalFilesExpected.ToArray());
+            return TestUpdateForProject(
+                dependencyName,
+                oldVersion,
+                newVersion,
+                projectContents,
+                expectedProjectContents,
+                additionalFiles: realizedAdditionalFiles.ToArray(),
+                additionalFilesExpected: realizedAdditionalFilesExpected.ToArray()
+            );
```

Comment:
Preference would be for the closing paren of argument and parameter lists to be on the same line as the last element. Despite not being enforceable by dotnet-format, this would match the existing style.

```suggestion
            return TestUpdateForProject(
                dependencyName,
                oldVersion,
                newVersion,
                projectContents,
                expectedProjectContents,
                additionalFiles: realizedAdditionalFiles.ToArray(),
                additionalFilesExpected: realizedAdditionalFilesExpected.ToArray());
```